rps.js

```diff
@@ -0,0 +1,27 @@
+const getUserChoice =() =>{
```

Comment:
"()=>" is an ES6 function declaration. it will be covered in unit 2.
The user input should be entered as an argument.
The program should run as a one function.
Also the desired output should tell the user who won.